sdk/keyvault/keyvault-admin/test/public/accessControlClient.spec.ts

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { assert } from "chai";
+import { env, Recorder } from "@azure/test-utils-recorder";
+
+import { KeyVaultAccessControlClient } from "../../src";
+import { authenticate } from "../utils/authentication";
+
+describe("KeyVaultAccessControlClient", () => {
+  let client: KeyVaultAccessControlClient;
+  let recorder: Recorder;
+  let generateFakeUUID: () => string;
+  const globalScope = "/";
+
+  beforeEach(async function() {
+    const authentication = await authenticate(this);
+    client = authentication.accessControlClient;
+    recorder = authentication.recorder;
+    generateFakeUUID = authentication.generateFakeUUID;
+  });
+
+  afterEach(async function() {
+    await recorder.stop();
+  });
+
+  // The tests follow
+
+  it("listRoleDefinitions", async function() {
+    const expectedType = "Microsoft.Authorization/roleDefinitions";
+    let receivedRoles: string[] = [];
+
+    for await (const roleDefinition of client.listRoleDefinitions("/")) {
+      // Each role definition will have the shape of:
+      //
+      //   {
+      //     id: 'Microsoft.KeyVault/providers/Microsoft.Authorization/roleDefinitions/<ID>',
+      //     name: '<ID>',
+      //     type: '<role-type>',
+      //     roleName: '<role-name>',
+      //     // ...
+      //   }
+      //
+      assert.equal(roleDefinition.type, expectedType);
+      receivedRoles.push(roleDefinition.roleName!);
+    }
+
+    assert.deepEqual(receivedRoles, [
```

Comment:
nit: these names have changed in the past - you may just want to validate that the received roles count is not 0